mojaloop/requirements.yaml

```diff
@@ -32,3 +32,13 @@ dependencies:
   version: 8.7.1
   repository: "file://../transaction-requests-service"
   condition: transaction-requests-service.enabled
+- name: finance-portal
+  version: 8.6.0
+  repository: "file://../finance-portal"
+  condtion: finance-portal.enabled
+- name: nginx-ingress
```

Comment:
Please remove the ingress controller.

This is a general k8s config, and should not be part of the mojaloop deployment.

It is up to the deployer on which Ingress controller they wish to utilise